src/storm/modelchecker/prctl/SparseDtmcPrctlModelChecker.cpp

```diff
@@ -101,8 +101,8 @@ std::unique_ptr<CheckResult> SparseDtmcPrctlModelChecker<SparseDtmcModelType>::c
         storm::modelchecker::helper::SparseDeterministicStepBoundedHorizonHelper<ValueType> helper;
         std::vector<ValueType> numericResult =
             helper.compute(env, storm::solver::SolveGoal<ValueType>(this->getModel(), checkTask), this->getModel().getTransitionMatrix(),
-                           this->getModel().getBackwardTransitions(), leftResult.getTruthValuesVector(), rightResult.getTruthValuesVector(),
-                           pathFormula.getNonStrictLowerBound<uint64_t>(), pathFormula.getNonStrictUpperBound<uint64_t>(), checkTask.getHint());
+                           leftResult.getTruthValuesVector(), rightResult.getTruthValuesVector(), pathFormula.getNonStrictLowerBound<uint64_t>(),
```

Comment:
@tquatmann is it clear why we referred to the backward transitions previously?

Reply:
They're used below the line where they are calculated now, in a call to `performProbGreater0`: https://github.com/moves-rwth/storm/blob/b26d9afa8c35a02d5d009f13bc3e912249cd4535/src/storm/modelchecker/helper/finitehorizon/SparseDeterministicStepBoundedHorizonHelper.cpp#L43 which is defined in graph.cpp: https://github.com/moves-rwth/storm/blob/b26d9afa8c35a02d5d009f13bc3e912249cd4535/src/storm/utility/graph.cpp#L322

This function iterates over the rows of the backwards transitions, as it essentially does a reverse flood-fill to find the states which can reach the target state(s). Using the non-transposed matrix would thus require iterating over a column, which is slow and I'm not sure it's even possible with how sparse matrices are implemented.

I don't know off of the top of my head if the same result can be achieved in a forwards manner, but I think it would be much slower since it would probably require `maximalSteps` iterations over all vertices.

This was the only use of it in the helper, I'm not sure if there are any other potential reasons to refer to the backward transitions but I doubt it.